portal-impl/test/unit/com/liferay/portal/kernel/util/SessionClicksTest.java

```diff
@@ -128,6 +132,23 @@ public void testPutMaxSizeTerms() {
 		Assert.assertNull(SessionClicks.get(httpSession, key, null));
 	}
 
+	private void _setUpPropsUtil(String key, String value) {
```

Comment:
@dantewang Please use `PropsTestUtil`

Reply:
@tinatian By using `PropsTestUtil`, I'll have to break what we discussed "only set the relevant property in each test method", because the proxy `PropsTestUtil` created doesn't fallback to portal's default values for properties not provided by us.

It would be something looks like:

```
@Test
public void testPutMaxAllowedValues() {
    PropsTestUtil.setProps(
        HashMapBuilder.<String, Object>put(
            PropsKeys.SESSION_CLICKS_MAX_ALLOWED_VALUES,
            String.valueOf(_MAX_ALLOWED_VALUES)
        ).put(
            PropsKeys.SESSION_CLICKS_MAX_SIZE_TERMS,
            propsImpl.get(PropsKeys.SESSION_CLICKS_MAX_SIZE_TERMS) // Not testing max size for key/values in this method so set the portal default value.
        ).build());
```